pkg/console/subresource/configmap/managed_cluster_oauth_server_cert.go

```diff
@@ -0,0 +1,41 @@
+package configmap
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/console/subresource/util"
+)
+
+func DefaultManagedClusterOAuthServerCertConfigMap(clusterName string, caBundle string, cr *operatorv1.Console) *corev1.ConfigMap {
```

Comment:
I cannot find where this function is used?

Reply:
https://github.com/TheRealJon/console-operator/blob/5c44b4725b3c6d3a76ffd517968d1bc6e45c00c1/pkg/console/controllers/managedclusters/controller.go#L346